packages/utils/telemetry-utils/src/index.ts

```diff
@@ -2,6 +2,15 @@
  * Copyright (c) Microsoft Corporation. All rights reserved.
  * Licensed under the MIT License.
  */
+
+import { ITelemetryBaseLogger } from "@fluidframework/common-definitions";
+
+//* Where should this actually go? common-definitions?
+declare global {
+    /** This function may be provided by the environment, like a mocha test hook */
+    export const getTestLogger: (() => ITelemetryBaseLogger) | undefined;
+}
+
```

Comment:
@anthony-murphy - Thoughts on where this declaration should go? common-definitions alongside `ITelemetryBaseLogger`? Or some other package?  Would it make sense to consolidate this and `getFluidTestDriver` in a package called something like `@fluid-test/test-seams`?

Reply:
i'm leaning towards test-driver-definitions. i actually debated just calling it test definitions, which may have been the right approach

Reply:
K makes sense to me, I'll rename the package.

Reply:
...in a future PR